utils/gyb_syntax_support/StmtNodes.py

```diff
@@ -12,12 +20,8 @@
 
     # while-stmt -> label? ':'? 'while' condition-list code-block ';'?
     Node('WhileStmt', kind='Stmt',
-         traits=['WithCodeBlock', 'Labeled'],
+         traits=['WithCodeBlock'],
```

Comment:
Should the specifications be updated as well?

Reply:
What specification are you referring to? The grammar is strictly correct - we don't necessarily want the notion of a "labeled statement" in the formal grammar since not all the statements support labels!

Reply:
I was referring to the comment for this specific node (and others you modified), not the actual grammar from the language reference, although, curiously, it does have a notion of a [labeled statement](https://docs.swift.org/swift-book/ReferenceManual/Statements.html#grammar_labeled-statement) too.